recipes/python/assets/create_oracle_asset.py

```diff
@@ -0,0 +1,247 @@
+# Cred? if doesn't already exist
+# register creds
+
+## The script can be run with Python 3.5 or higher version.

```

Comment:
FYI, Python 3.5 reached EOL over three years ago.

Reply:
Yeah I'm not sure if we want to update the statement at the top to what it was developed with? It'll probably run with 3.5 but yeah I get its outdated.